src/core/components/hv-route/index.tsx

```diff
@@ -556,7 +556,7 @@ function RouteFC(props: Types.FCProps) {
   return (
     <HvRouteInner
       // eslint-disable-next-line react/jsx-props-no-spreading
-      {...{ ...props, navigationService, navigator, needsLoad, onUpdate }}
+      {...{ ...props, navigationService, navigator, needsLoad }}
```

Comment:
does `HvRouteInner` end up having access to the `onUpdate` via another mean?

Reply:
The navigator will, which is handled in a separate PR. Any screens generated in the `HvRouteInner` will create their own `onUpdate`